imake.c

```diff
@@ -501,7 +501,7 @@ LogFatal(x0,x1)
 	char *x0, *x1;
 {
 #ifndef WIN32
-	extern char	*sys_errlist[];
+	extern const char *const sys_errlist[];
```

Comment:
per perror(3) direct access to `sys_errlist` is deprecated in favor of `strerror()`.  It looks like there's only one site to change, if you had interest in doing that.  If not, that's okay, too.

Thanks for submitting these fixes.